source/index.html

```diff
@@ -1,12 +1,135 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
+    <title>Кэт энерджи</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="mein-header__logo" href="#">
```

Comment:
Ошибка в именовании класса по БЭМ:
```suggestion
      <a class="main-header__logo" href="#">
```
Проверить правильность именования можно в этом [сервисе](https://yoksel.github.io/html-tree/).